core/templates/pages/admin-page/roles-tab/admin-roles-tab.directive.ts

```diff
@@ -360,6 +360,10 @@ angular.module('oppia').directive('adminRolesTab', [
             $rootScope.$apply();
           });
         };
+
+        ctrl.onViewContributorMethodChange = function() {
+          ctrl.contributionReviewersDataFetched = false;
+        };
```

Comment:
```suggestion

        ctrl.clearReviewersData = function() {
          ctrl.contributionReviewersDataFetched = false;
          ctrl.result = {};
        };
```
This will help us to have a better function name as it explains what it's doing and in the template ng-change="clearReviewersData" will make it easier for anyone to understand.

Reply:
done!